### ISSUE TITLE
VoiceOver on Mac keeps saying `row 1 collapsed`

### ISSUE THREAD

**Issue by jrieken** [Microsoft/vscode#33331]
testing #32858

* on mac, enable voice over
* select a collapsed folder in the explorer
* it endlessly keeps saying `row 1 collapsed`, it doesn't say the name of the folder

<img width="1217" alt="screen shot 2017-08-29 at 10 38 29" src="https://user-images.githubusercontent.com/1794099/29812324-7544872a-8ca6-11e7-931f-274df89be9de.png">

**Comment by bpasero** [comment]
I cannot reproduce the endless VoiceOver (just once). Seems related to https://github.com/Microsoft/vscode/issues/10135

ARIA wise the node is a `role="treeitem"` with `aria-expanded="false"`

![image](https://user-images.githubusercontent.com/900690/29817163-33ffb026-8cb7-11e7-8d07-b99e156fab34.png)

If someone has a clue how to change this on macOS, help welcome.

**Comment by joaomoreno** [comment]
To keep the number of issues in our inbox on a manageable level, we're closing issues that have been on the backlog for a long time but haven't gained traction: We look at the number of votes the issue has received and the number of duplicates issues filed.

Thanks for your understanding. Happy coding!
